runtime/include/chpl-gpu-impl.h

```diff
@@ -63,6 +69,14 @@ void chpl_gpu_impl_set_peer_access(int dev1, int dev2, bool enable);
 
 void chpl_gpu_impl_use_device(c_sublocid_t dev_id);
 
+// TODO this should return true for NVIDIA, too. The current NVIDIA
```

Comment:
It should go away -- it was an interim comment.

For more background re allocators: CUDA and HIP have what is called "stream-ordered memory allocators" That's basically `cuMallocAsync` and `cuFreeAsync` and some support for memory pools for faster alloc/dealloc if you keep doing that repeatedly. I started with using them (not pools just async allocation/free), and back then the implementation for this support function was more complicated. I backed away from it because I hit an issue with the nvlink test that I can't explain quickly. With us back to not using stream-ordered memory allocators, this support function should just return true for `nvidia`. And that's what that scribble of a comment tries to convey :) It does just return true at the moment.